unit-tests/shared/src/test/scala/javalib/lang/ThrowablesTest.scala

```diff
@@ -22,6 +24,7 @@ class ThrowablesTest {
       throwable: Throwable,
       expectedCause: Throwable
   ): Unit = {
+    assumeFalse(Platform.asanEnabled)
```

Comment:
Similarry to to is32BitPlatform this should also be moved to dedicated assumption method with meaningful message